__tests__/app.test.js

```diff
@@ -71,4 +71,32 @@ describe('Get /api', () => {
 
     })
 
+})
+
+describe('/api/articles', () => {
+
+    describe('Get /api/articles/:article_id', () => {
+
+        test('Returns an article by its id and has the correct properties', () => {
+
+            return request(app)
+                .get('/api/articles/3')
+                .expect(200)
+                .then(({ body }) => {
+
+                    expect(body).toHaveProperty('author')
+                    expect(body).toHaveProperty('title')
+                    expect(body).toHaveProperty('article_id')
```

Comment:
Would you expect this article to have just any number? You can be more specific here with the article_id you expect this article to have